snap/snapcraft.yaml

```diff
@@ -130,6 +110,18 @@ parts:
       - libncursesw6
       - python3-tk
 
+  oudedetai-dump:
+    source: https://github.com/FaithLife-Community/LogosLinuxInstaller.git
+    plugin: dump
+    organize:
+      snap/bin/run.sh: bin/run.sh
+      ou_dedetai/assets: assets/
+      ou_dedetai/img: img/
+    stage:
+      - bin/run.sh
+      - assets/
+      - img
+
```

Comment:
This is a nice simplification. If we're making these changes, might as well update the repo name to use "OuDedetai".

Reply:
We can do that later, we're releasing now and it'd be nice to have this. We need to change the repo name in other places as well